tyssue/solvers/viscous.py

```diff
@@ -81,7 +81,7 @@ def set_pos(self, pos):
         return self._set_pos(self.eptm, self.geom, pos)
 
     def record(self, t):
-        self.history.record(["vert"], t)
+        self.history.record(["vert", "face", "edge"], t)
```

Comment:
At first, I only record vertices at every time point and face and edge only if topology changes, because all the geometry is contained in the vertices positions. Is there a reason to revert this mechanism?

Reply:
At the beginning, I had some problem with History class... :/ Some understanding /reading problem...
I forgot to revert this change.

Reply:
Can I choose to record face and edge also at every time ? Not only when the topo change.

Reply:
sure!

Reply:
how ? Is it an option in History ? 
I feel like I can't understand code anymore...

Reply:
Oh I thought you were asking if you can implement the option :smile: !!

So for now you can't, we'd need to implement it. We can look into that later!

For the record, I think you understand the code very well :wink:

Reply:
Oh, I was afraid that the option was present and clear to understand but can't see it... :)
Okay, so for me I keep the line `self.history.record(["vert", "face", "edge"], t)`, because I need some information in face_df during analysis, or viewing.

Reply:
yeah you're right, the space saving is probably not worth the added complexity. Let's leave it with your version